diff --git a/node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js b/node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js
index 3756b08..433a428 100644
--- a/node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js
+++ b/node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js
@@ -172,6 +172,7 @@ class MessageQueue {
     onFail: ?Function,
     onSucc: ?Function,
   ): any {
+    /*
     if (__DEV__) {
       invariant(
         global.nativeCallSyncHook,
@@ -183,6 +184,8 @@ class MessageQueue {
     }
     this.processCallbacks(moduleID, methodID, params, onFail, onSucc);
     return global.nativeCallSyncHook(moduleID, methodID, params);
+    */
+    return [];
   }
 
   processCallbacks(
